packages/react-components/src/dot/src/Dot.tsx

```diff
@@ -65,6 +66,22 @@ export function InnerDot(props: InnerDotProps) {
     );
 }
 
+function useColor(color: string) {
+    return useMemo(() => {
+        if (color) {
+            if (color.startsWith("rgb") || color.startsWith("hsl") || color.startsWith("#")) {
+                return color;
+            } else if (color.startsWith("--")) {
+                return `var(${color})`;
+            } else {
+                return `var(--o-ui-global-${color})`;
+            }
+        } else {
+            return;
+        }
```

Comment:
you don't need an `else` to specify a default return.

```
return useMemo(() => {
     if (color) {
            if (color.startsWith("rgb") || color.startsWith("hsl") || color.startsWith("#")) {
                return color;
            } else if (color.startsWith("--")) {
                return `var(${color})`;
            } else {
                return `var(--o-ui-global-${color})`;
            }
    }
    
    return undefined;
}
```

Reply:
In fact you could probably even omit `return undefined;` since returning nothing if the same but I think eslint or TS might complain if you do omit the default return.